ansible_collections/arista/avd/molecule/anta_runner/anta/avd_catalogs/allow_bgp_vrfs_run/dc2-leaf3a.arista.com.json

```diff
@@ -51,26 +51,6 @@
             "port": "Ethernet15",
             "neighbor_device": "dc2-leaf2b",
             "neighbor_port": "Ethernet2"
-          },
-          {
-            "port": "Ethernet16",
-            "neighbor_device": "dc2-leaf3b.arista.com",
-            "neighbor_port": "Ethernet16"
-          },
-          {
-            "port": "Ethernet17",
-            "neighbor_device": "dc2-leaf3b.arista.com",
-            "neighbor_port": "Ethernet17"
-          },
-          {
-            "port": "Ethernet18",
-            "neighbor_device": "dc2-leaf3b.arista.com",
-            "neighbor_port": "Ethernet18"
-          },
-          {
-            "port": "Ethernet19",
-            "neighbor_device": "dc2-leaf3b.arista.com",
-            "neighbor_port": "Ethernet19"
```

Comment:
should 17, 18 and 19 one be going? Feels like we are only shutting down 16 in your host_vars

Reply:
my bad - by default ports are shutdown on leaf3b and so thats why.